facades/PC/src/main/java/org/terasology/engine/TerasologyCommand.java

```diff
@@ -0,0 +1,417 @@
+        }
+
+        return 0;
+    }
+
+    private static String getNewTitle(String title) {
+        String newTitle = title.substring(0, getPositionOfLastDigit(title));
+        int fileNumber = getLastNumber(title);
+        fileNumber++;
+        return (newTitle + " " + fileNumber);
+    }
+
+    private static SplashScreen configureSplashScreen() {
+        int imageHeight = 283;
+        int maxTextWidth = 450;
+        int width = 600;
+        int height = 30;
+        int left = 20;
+        int top = imageHeight - height - 20;
+
+        Rectangle rectRc = new Rectangle(left, top, width, height);
+        Rectangle textRc = new Rectangle(left + 10, top + 5, maxTextWidth, height);
+        Rectangle boxRc = new Rectangle(left + maxTextWidth + 10, top, width - maxTextWidth - 20, height);
+
+        SplashScreenBuilder builder = new SplashScreenBuilder();
+
+        String[] imgFiles = new String[] {
+                "splash_1.png",
+                "splash_2.png",
+                "splash_3.png",
+                "splash_4.png",
+                "splash_5.png"
+        };
+
+        Point[] imgOffsets = new Point[] {
+                new Point(0, 0),
+                new Point(150, 0),
+                new Point(300, 0),
+                new Point(450, 0),
+                new Point(630, 0)
+        };
+
+        EngineStatus[] trigger = new EngineStatus[] {
+                TerasologyEngineStatus.PREPARING_SUBSYSTEMS,
+                TerasologyEngineStatus.INITIALIZING_MODULE_MANAGER,
+                TerasologyEngineStatus.INITIALIZING_ASSET_TYPES,
+                TerasologyEngineStatus.INITIALIZING_SUBSYSTEMS,
+                TerasologyEngineStatus.INITIALIZING_ASSET_MANAGEMENT,
+        };
+
+        try {
+            for (int index = 0; index < 5; index++) {
+                URL resource = TerasologyCommand.class.getResource("/splash/" + imgFiles[index]);
+                builder.add(new TriggerImageOverlay(verifyNotNull(resource, "resource /splash/%s", imgFiles[index]))
+                        .setTrigger(trigger[index].getDescription())
+                        .setPosition(imgOffsets[index].x, imgOffsets[index].y));
+            }
+
+            builder.add(new ImageOverlay(TerasologyCommand.class.getResource("/splash/splash_text.png")));
+
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        SplashScreen instance = builder
+                .add(new RectOverlay(rectRc))
+                .add(new TextOverlay(textRc))
+                .add(new AnimatedBoxRowOverlay(boxRc))
+                .build();
+
+        return instance;
+    }
+
+    private static void setupLogging() {
+        Path path = PathManager.getInstance().getLogPath();
+        if (path == null) {
+            path = Paths.get("logs");
+        }
+
+        LoggingContext.initialize(path);
+    }
+
+
+    private static void printUsageAndExit() {
+        // TODO: Add all this stuff back in to the help text.
```

Comment:
This is then added via the picocli library, isn't it? I don't think we need to copy word-by-word, but just make sure that users can figure out what options/flags are available.